core/catkin_ws/src/custom_msgs/msg/Object.msg

```diff
@@ -0,0 +1,10 @@
+std_msgs/Header header
+
+# Max/min x- and y-coordinates for the bounding box
+float64 xmin
+float64 ymin
+float64 xmax
+float64 ymax
+
```

Comment:
Add a comment here that these are for the buoys (which they are I think?)

Reply:
Those coordinates aren't just for the buoys -- they're meant to be a general form for every type of object. The label on line 9 will specify if we're looking at an alien buoy, gate, etc.

Reply:
Oh I see, this is generic, for any object cv recognizes